tests/components/here_weather/test_config_flow.py

```diff
@@ -0,0 +1,172 @@
+"""Tests for the here_weather config_flow."""
+from unittest.mock import patch
+
+import herepy
+
+from homeassistant.components.here_weather.const import DOMAIN, HERE_API_KEYS
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_NAME,
+    CONF_SCAN_INTERVAL,
+    EVENT_HOMEASSISTANT_START,
+)
+
+from . import mock_weather_for_coordinates
+
+from tests.common import MockConfigEntry
+
+
+async def test_config_flow(hass):
+    """Test we can finish a config flow."""
+    with patch(
+        "herepy.DestinationWeatherApi.weather_for_coordinates",
+        side_effect=mock_weather_for_coordinates,
+    ):
```

Comment:
We normally patch the integration setup function on all create entry results to avoid setting up the whole integration. See the scaffold test template for example.

Reply:
Done